autrainer/datasets/dcase_2018_t3.py

```diff
@@ -56,6 +57,10 @@ def __init__(
             file_type: File type of the features.
             file_handler: File handler to load the data.
             batch_size: Batch size.
+            features_path: Root path to features. Useful
+                when features need to be extracted and store
```

Comment:
...  when features need to be extracted and store`d` ... (for all new docstrings)

Reply:
de5c9b1